adr/template.md

```diff
@@ -0,0 +1,36 @@
+# TITLE
+
+Date: 20xx-xx-xx
```

Comment:
I'm concerned that this doesn't leave us the ability to expand our work into the 22nd century.